### PR TITLE
Track disabling analyzers in `analyzer.log`

### PR DESCRIPTION
**Pull request by J-Gras**
This adds an option to track disabling of analyzers in `analyzer.log`.

### CODE REVIEW

File: testing/btest/Baseline/scripts.base.frameworks.analyzer.logging/analyzer.log-default [hunk 1]
```diff
@@ -13,6 +13,4 @@ XXXXXXXXXX.XXXXXX	violation	protocol	DCE_RPC	ClEkJM2Vm5giqnMf4h	-	10.0.0.55	5399
 XXXXXXXXXX.XXXXXX	violation	protocol	DCE_RPC	ClEkJM2Vm5giqnMf4h	-	10.0.0.55	53994	60.190.189.214	8124	Binpac exception: binpac exception: &enforce violation : DCE_RPC_Header:rpc_vers	-
 XXXXXXXXXX.XXXXXX	violation	protocol	DCE_RPC	ClEkJM2Vm5giqnMf4h	-	10.0.0.55	53994	60.190.189.214	8124	Binpac exception: binpac exception: &enforce violation : DCE_RPC_Header:rpc_vers	-
 XXXXXXXXXX.XXXXXX	violation	protocol	DCE_RPC	ClEkJM2Vm5giqnMf4h	-	10.0.0.55	53994	60.190.189.214	8124	Binpac exception: binpac exception: &enforce violation : DCE_RPC_Header:rpc_vers	-
-XXXXXXXXXX.XXXXXX	violation	protocol	DCE_RPC	ClEkJM2Vm5giqnMf4h	-	10.0.0.55	53994	60.190.189.214	8124	Binpac exception: binpac exception: &enforce violation : DCE_RPC_Header:rpc_vers	-
```
Comment (awelzel) [inline testing/btest/Baseline/scripts.base.frameworks.analyzer.logging/analyzer.log-default:16]: Is this related to the `&priority` change? Could you leave a comment what's going on here?

Btw, if you rebase the OSX failures should go (will only test on sonoma)

Comment (awelzel) [reply testing/btest/Baseline/scripts.base.frameworks.analyzer.logging/analyzer.log-default:16]: Uh - thanks! A comment would be nice. Not sure why that analyzer does that, but that's a different story.

Thanks.

**Review by awelzel — APPROVED** [review]
@timwoj / @ckreibich - I'd merge this (also for 7.0). How strong do you feel about waiting until we forked?

**Comment by timwoj** [comment]
I don't see anything wrong with dragging this into 7.0.

**Pull request merged**